src/astx/transpilers/python.py

```diff
@@ -255,3 +255,23 @@ def visit(self, node: astx.LiteralComplex64) -> str:
         real = node.value.real
         imag = node.value.imag
         return f"complex({real}, {imag})"
+
+    @dispatch  # type: ignore[no-redef]
+    def visit(self, node: astx.UTF8String) -> str:
+        """Handle UTF8String nodes."""
+        return f'"{node.value}"'
```

Comment:
I think you need to use repr here and let python to decide withc quotation mark it will use, so it can handle different combinations like: `" 'a' "` and `' "a" '`

```suggestion
        return repr(node.value)
```